diff --git a/node_modules/react-use/esm/useAsyncFn.js b/node_modules/react-use/esm/useAsyncFn.js
index 01d7307..af77317 100644
--- a/node_modules/react-use/esm/useAsyncFn.js
+++ b/node_modules/react-use/esm/useAsyncFn.js
@@ -21,6 +21,7 @@ export default function useAsyncFn(fn, deps, initialState) {
             return value;
         }, function (error) {
             isMounted() && callId === lastCallId.current && set({ error: error, loading: false });
+            console.error('error in useAsync*', error)
             return error;
         });
     }, deps);
diff --git a/node_modules/react-use/lib/useAsyncFn.js b/node_modules/react-use/lib/useAsyncFn.js
index e06fd81..d341856 100644
--- a/node_modules/react-use/lib/useAsyncFn.js
+++ b/node_modules/react-use/lib/useAsyncFn.js
@@ -23,6 +23,7 @@ function useAsyncFn(fn, deps, initialState) {
             return value;
         }, function (error) {
             isMounted() && callId === lastCallId.current && set({ error: error, loading: false });
+            console.error('error in useAsync*', error)
             return error;
         });
     }, deps);
